Library/Homebrew/test/cask/pkg_spec.rb

```diff
@@ -6,34 +6,6 @@
     let(:empty_response) { double(stdout: "", plist: { "volume" => "/", "install-location" => "", "paths" => {} }) }
     let(:pkg) { described_class.new("my.fake.pkg", fake_system_command) }
 
-    it "removes files and dirs referenced by the pkg" do
```

Comment:
@homebrew/cask: FYI that this test has been removed. Link in the PR body to example of a false positive/flake from it.